test/module/libs/amount/amount_test.cpp

```diff
@@ -35,26 +33,28 @@ TEST_F(AmountTest, TestBasic) {
   auto c = a + b;
   ASSERT_EQ(c.to_string(), "1.84");
 
-  auto d = a.percentage(c);
+  auto d = a.percentage(c); // taking 1.84% of 1.23
   ASSERT_EQ(d.to_string(), "0.02");
 
-  auto e = a.percentage(amount::Amount(256)); // taking 256% of the amount == 3.1488
+  auto e = a.percentage(iroha::Amount(256)); // taking 256% of the amount == 3.1488
```

Comment:
You can add cast operator, so you can call just `a.percentage(256)`

Reply:
To do so, we have `percentage(uint256_t percents)`

Reply:
Ah,ok, missed this